### ISSUE TITLE
A11y - Issue Image missing a text alternative

### ISSUE THREAD

**Issue by davidstenger** [GSA/search-gov-website#309]
Making the big move: Search in support of federal web modernization | Search.gov
Page URL: https://search.gov/about/updates/year-in-review/special-report/overview.html

![Image](https://github.com/user-attachments/assets/6114b200-0d7e-4fca-b318-6a4eedf85096)

All non-decorative images must have a text alternative (also known as "alt text").

Text alternatives are used to describe images to people who are unable to see them. This includes people who use screen readers to access the web.

The text alternative should describe the image within the context of the page — letting visitors know what the image means and why it was included.

WCAG 2.2: [Understanding Success Criterion 1.1.1: Non-text Content](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content)

Add a text alternative for the image.
Using the alt attribute.
`<img alt="" />`
